lib/seccomp/CSystemCallFilter_Linux.cc

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+#include "seccomp/CSystemCallFilter.h"
+
+#include <core/CLogger.h>
+
+#include <linux/audit.h>
+#include <linux/filter.h>
+#include <sys/prctl.h>
+#include <sys/syscall.h>
+
+
+namespace ml {
+namespace seccomp {
+
+// The old x32 ABI always has bit 30 set in the sys call numbers.
+// The x64 architecture should fail these calls
+unsigned int UPPER_NR_LIMIT = 0x3FFFFFFF;
```

Comment:
I think this and the two below should be `const std::uint32_t` (plus `#include <cstdint>`).  The struct member they're used with is explicitly 32 bits, and whilst `unsigned int` is 32 bits on all the compilers we're currently using it's best practice to specify the size exactly where the consuming code does.